packages/tracing/src/integrations/mongo.ts

```diff
@@ -0,0 +1,211 @@
+  'createIndexes', // createIndexes(indexSpecs, options, callback)
+  'deleteMany', // deleteMany(filter, options, callback)
+  'deleteOne', // deleteOne(filter, options, callback)
+  'distinct', // distinct(key, query, options, callback)
+  'drop', // drop(options, callback)
+  'dropIndex', // dropIndex(indexName, options, callback)
+  'dropIndexes', // dropIndexes(options, callback)
+  'estimatedDocumentCount', // estimatedDocumentCount(options, callback)
+  'findOne', // findOne(query, options, callback)
+  'findOneAndDelete', // findOneAndDelete(filter, options, callback)
+  'findOneAndReplace', // findOneAndReplace(filter, replacement, options, callback)
+  'findOneAndUpdate', // findOneAndUpdate(filter, update, options, callback)
+  'indexes', // indexes(options, callback)
+  'indexExists', // indexExists(indexes, options, callback)
+  'indexInformation', // indexInformation(options, callback)
+  'initializeOrderedBulkOp', // initializeOrderedBulkOp(options, callback)
+  'insertMany', // insertMany(docs, options, callback)
+  'insertOne', // insertOne(doc, options, callback)
+  'isCapped', // isCapped(options, callback)
+  'mapReduce', // mapReduce(map, reduce, options, callback)
+  'options', // options(options, callback)
+  'parallelCollectionScan', // parallelCollectionScan(options, callback)
+  'rename', // rename(newName, options, callback)
+  'replaceOne', // replaceOne(filter, doc, options, callback)
+  'stats', // stats(options, callback)
+  'updateMany', // updateMany(filter, update, options, callback)
+  'updateOne', // updateOne(filter, update, options, callback)
+] as const;
+
+const OPERATION_SIGNATURES: {
+  [op in Operation]?: string[];
+} = {
+  bulkWrite: ['operations'],
+  countDocuments: ['query'],
+  createIndex: ['fieldOrSpec'],
+  createIndexes: ['indexSpecs'],
+  deleteMany: ['filter'],
+  deleteOne: ['filter'],
+  distinct: ['key', 'query'],
+  dropIndex: ['indexName'],
+  findOne: ['query'],
+  findOneAndDelete: ['filter'],
+  findOneAndReplace: ['filter', 'replacement'],
+  findOneAndUpdate: ['filter', 'update'],
+  indexExists: ['indexes'],
+  insertMany: ['docs'],
+  insertOne: ['doc'],
+  mapReduce: ['map', 'reduce'],
+  rename: ['newName'],
+  replaceOne: ['filter', 'doc'],
+  updateMany: ['filter', 'update'],
+  updateOne: ['filter', 'update'],
+};
+
+interface MongoCollection {
+  collectionName: string;
+  dbName: string;
+  namespace: string;
+  prototype: {
+    [operation in Operation]: (...args: unknown[]) => unknown;
+  };
+}
+
+interface MongoOptions {
+  operations?: Operation[];
+  describeOperations?: boolean | Operation[];
+}
+
+/** Tracing integration for node-postgres package */
+export class Mongo implements Integration {
+  /**
+   * @inheritDoc
+   */
+  public static id: string = 'Mongo';
+
+  /**
+   * @inheritDoc
+   */
+  public name: string = Mongo.id;
+
+  private _operations: Operation[];
+  private _describeOperations?: boolean | Operation[];
```

Comment:
Is "describe" your word or theirs? If it's yours, I wonder if we might name this option differently, to a) differentiate it from the `description` field in a transaction, and b) indicate that what it's really about is recording the operation's inputs.

Reply:
I think `describe` is used in database lingo.